recipes/basemap-data-hires/meta.yaml

```diff
@@ -1,4 +1,4 @@
-package:
+ package:
```

Comment:
After all, we don't want to muck with it while it tries to remove it post-feedstocking.